### PR TITLE
Add CLASSPATH for Windows

### PR DESCRIPTION
**Pull request by MengMeng96**
## Description

The variable classpath in ts/model_server.py is not suitable for Windows. Add suitable classpath for Windows.

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Feature/Issue validation/testing

Please describe the tests [UT/IT] that you ran to verify your changes and relevent result summary. Provide instructions so it can be reproduced. 
Please also list any relevant details for your test configuration.

- UT/IT execution results

- Logs

## Checklist:

- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] New and existing unit tests pass locally with these changes?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Comment by gunandrose4u** [comment]
looks goof to me

**Comment by nbcsm** [comment]
https://github.com/pytorch/serve/issues/582

File: ts/model_server.py [hunk 1]
```diff
@@ -80,8 +81,13 @@ def start():
...
             ts_conf_file = ts_config
 
-        class_path = \
-            ".:{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        if(platform.system() == 'Windows'):
+            class_path = \
+                        "{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        else:
+            class_path = \
+                        ".:{}".format(os.path.join(ts_home, "ts/frontend/*"))
```
Comment (harshbafna) [inline ts/model_server.py:89]: Both the strings are more or less the same except for a couple of characters, better create a common string before if/else block and format it based on the platform.

Comment (MengMeng96) [reply ts/model_server.py]: Thanks for you comment. I have modified the code.

File: ts/model_server.py [hunk 2]
```diff
@@ -80,8 +81,9 @@ def start():
                 sys.exit(1)
             ts_conf_file = ts_config
 
-        class_path = \
-            ".:{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        class_path = "{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        if platform.system() != 'Windows':
```
Comment (harshbafna) [inline ts/model_server.py:85]: platform_path_saparator = {
    "Windows": "",
    "Darwin": ".:",
    "Linux": ".:"
}

class_path = "{}{}".format(platform_path_saparator[platform.system()], os.path.join(ts_home, "ts/frontend/*"))

How about this?

Comment (MengMeng96) [reply ts/model_server.py]: Sure, I have modified the code.

Comment (MengMeng96) [reply ts/model_server.py]: @harshbafna I have modified the code, please check it.

Comment (harshbafna) [reply ts/model_server.py]: LGTM. thanks.

**Pull request merged**